File: backend/components/metiers/administration_publique/guides/fallback/accessibility/fallback_accessibility.test.js
// fallback_accessibility.test.js – Tests unitaires et edge cases JS
const { fallbackAccessibility } = require('./fallback_accessibility');

test('fallbackAccessibility structure', () => {
  expect(typeof fallbackAccessibility).toBe('object');
  expect(fallbackAccessibility).toHaveProperty('id');
  expect(fallbackAccessibility).toHaveProperty('description');
});

test('fallbackAccessibility status', () => {
  expect(['compliant', 'partial', 'non-compliant']).toContain(fallbackAccessibility.status);
});

File: backend/components/metiers/administration_publique/guides/fallback/accessibility/fallback_accessibility.test.py
# fallback_accessibility.test.py – Tests unitaires et edge cases Python
from .fallback_accessibility import fallback_accessibility

def test_fallback_accessibility_structure():
    assert isinstance(fallback_accessibility, dict)
    assert 'id' in fallback_accessibility
    assert 'description' in fallback_accessibility

def test_fallback_accessibility_accessible():
    assert fallback_accessibility.get('status') in ['compliant', 'partial', 'non-compliant']
